h2o-core/src/main/java/water/H2O.java

```diff
@@ -442,11 +443,19 @@ else if (s.matches("flatfile")) {
       }
       else if (s.matches("port")) {
         i = s.incrementAndCheck(i, args);
-        trgt.port = s.parseInt(args[i]);
+        int portNum = s.parseInt(args[i]);
```

Comment:
This could be a method on OptString (eg. parsePort), that would be more readable + no code duplicities.